app/javascript/src/containers/Dashboard.jsx

```diff
@@ -15,6 +15,7 @@ class Dashboard extends Component {
         <NavBar />
         <div className="body">
           Dashboard
+          <Test />
```

Comment:
```suggestion
```

Remove this as we should not be using this component.

Reply:
deleted